tools/server/webui/src/lib/stores/server.svelte.ts

```diff
@@ -52,6 +55,10 @@ class ServerStore {
 	private _error = $state<string | null>(null);
 	private _serverWarning = $state<string | null>(null);
 	private _slotsEndpointAvailable = $state<boolean | null>(null);
+	private _serverMode = $state<ServerMode | null>(null);
+	private _selectedModel = $state<string | null>(null);
+	private _availableModels = $state<ApiRouterModelMeta[]>([]);
+	private _modelLoadingStates = $state<Map<string, boolean>>(new Map());
```

Comment:
Need to rethink this if it should go to a separate service + store (`router`/`models`)